src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
+
+  switch (true) {
+    case days >= 3 && days < 7: {
+      return rentalCost - 20;
+    }
+
+    case days >= 7: {
+      return rentalCost - 50;
```

Comment:
Avoid using hardcoded values for the discount amounts and the number of days that qualify for a discount. Define constants with descriptive names to represent these values.